src/applications/disability-benefits/all-claims/tests/all-claims-wizard.cypress.spec.js

```diff
@@ -105,7 +105,7 @@ describe('526 wizard', () => {
       value: 'yes-bdd',
     });
 
-    cy.get('va-date').should('exist');
+    cy.get('va-daatee').should('exist');
```

Comment:
Not sure what this is...

Reply:
Just forcing a failure to test something.